tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -27,4 +30,27 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 			Expect(stdOut).To(ContainSubstring("etcdoperator"))
 		})
 	})
+
+	Context("When creating an operator backed service", func() {
+		It("should be able to create EtcdCluster from its alm example", func() {
+			// First let's grab the etcd operator's name from "odo catalog list services" output
+			operators := helper.CmdShouldPass("odo", "catalog", "list", "services")
+			etcdOperator := regexp.MustCompile(`etcdoperator\.*[a-z][0-9]\.[0-9]\.[0-9]`).FindString(operators)
+
+			helper.CmdShouldPass("odo", "service", "create", etcdOperator, "--crd", "EtcdCluster")
```

Comment:
Delete the service at the end of the spec to make sure odo service delete works in this scenario too.

Reply:
Currently we're doing this by `helper.CmdShouldPass("oc", "delete", "EtcdCluster", "example")` since there's no way to delete operator backed services right now. This reminds me that I need to create a user story for service deletion.

Reply:
Created #2792 to track this.

Reply:
Cool!!!